src/_playground/App.scss

```diff
@@ -96,12 +95,28 @@ $library-color: #61dafb;
             }
         }
 
+        @media (max-width: $mediaMaxWidth) {
+            .frDocs-Sidebar.sidebar-hidden{
```

Comment:
We should probably get stylelint setup on this repo, but there should be a space before the `{`.